core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1316,13 +1316,14 @@ public void run() {
             try {
               serviceConfigInterceptor.handleUpdate(serviceConfig);
               if (retryEnabled) {
-                throttle = getThrottle(config);
+                throttle = ServiceConfigUtil.getThrottlePolicy(
+                    config.get(GrpcAttributes.NAME_RESOLVER_SERVICE_CONFIG));
               }
-            } catch (RuntimeException re) {
+            } catch (Exception e) {
```

Comment:
Move the `//TODO` in the doc of `getThrottlePolicy` here. The //TODO is not only for `getThrottlePolicy`, but also for `getDouble()`, `getList()` etc.

Reply:
I removed the TODO since I got rid of MalformedConfigException.